book/website/project-design/pd-bias/pd-bias-confounders.md

```diff
@@ -0,0 +1,26 @@
+(pd-bias-confounders)=
+# Confounding Variables
+Correlation does not imply causation - if we want to know whether a relationship between an exposure and outcome is causal, we must first consider whether any other factors could have had an influence.
+
+Confounders are external factors which cause both the exposure and outcome.
+For example, there might be a strong correlation between ice cream sales and shark attacks at the beach, but there is no causal relationship here - both of these are more common in warm weather, so the weather is the confounder. 
+To measure the strength of any causal relationship between exposure and outcome, the strength of the relationship between the confounder and the outcome must also be measured by including the confounder as a model input. 
+In a simple model the strength of these effects ($x_1, x_2$) could be esimated as:
+$Outcome = x_1 \times  Exposure  + x_2 \times Confounder$
+
+
+In the study of causal inference it is common to use directed acyclic graphs (DAGs) to map the causal relationships between variables, which can help to identify confounders. 
+This can be helpful as it is not always clear whether a potential confounder is in fact a confounder, or a *mediator*, a *proxy confounder*, or a *competing exposure*. 
+
+![DAG_Example](https://hackmd.io/_uploads/H1YsrR6B0.png)
+[caption - A directed acyclic graph (DAG), with the relationship of interest indicated in red.]
```

Comment:
If you look on the deploy preview the caption and the alt text aren't pulling through- I think you need to add the figure into the directory: https://book.the-turing-way.org/community-handbook/style/style-figures.html